airbyte-config/models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -100,6 +100,7 @@ public class EnvConfigs implements Configs {
   private static final String CONTAINER_ORCHESTRATOR_IMAGE = "CONTAINER_ORCHESTRATOR_IMAGE";
   private static final String DD_AGENT_HOST = "DD_AGENT_HOST";
   private static final String DD_DOGSTATSD_PORT = "DD_DOGSTATSD_PORT";
+  private static final String DATA_PLANE_NAME = "DATA_PLANE_NAME";
```

Comment:
nice, this is almost exactly what I was thinking of.

I had thought of this in my head as two variables `REGION` and `IAAS` however, I think we get more flexibility with support for any name.  A single name also is easier for MVP